clients/go/apiban-ipsets/config.ini

```diff
@@ -0,0 +1,42 @@
+; Policy Agent configuration file
+;; Agent general settings
+; SSO token
+TOKEN=""
+; lk id
+LKID="0"
+; apiban protocol version
+VERSION=2
+; URL of white/blacklisted entries in DB
+URL="https://siem.intuitivelabs.com/api/"
+; log file or - for stdout
+log="/var/log/apiban-ipsets.log"
+; interval for the list refresh
+INTERVAL="60s"
```

Comment:
```
$ ./apiban-ipsets -l - -c config.ini 
2021/09/30 13:02:02 ** client start
loading config file "config.ini" ...
2021/09/30 13:02:02 blacklist ttl under 1s in "config.ini": 0s. Setting it to 0
```
`INTERVAL` does not exist yet; `TICK` is still used. I can change it later on.

Reply:
thx! fixed by 2330a52537a0ecd1c60c7b9618b9951b0c9b03be